EIPS/payoutable-token-draft.md

```diff
@@ -0,0 +1,302 @@
+
+In general, it is not advisable to place explicit routing instructions for the payouts on a verbatim basis on the blockchain, and it is advised to use a private communication alternatives. such as private channels, encrypted storage or similar,  to do so (external to the blockchain ledger). Another (less desirable) possibility is to place these instructions on the instructions field on encrypted form.
+
+
+## Motivation
+Nowadays most of the token payout requests,  a previous centralized transaction, to be able to define the payout destination to be able to execute the payout (burn transaction).
+In the aim of trying step by step to bring all the needed steps into decentralization, exposing all the needed steps of token lifecycle and payment transactions, a payout request can allow wallet owner to initiate the payout order via blockchain.
+Key benefits:
+
+* Payout, burning  traceability is enhanced bringing the initation into the ledger. All payment, payout statuses can be stored on chain.
+* Almost all money/token lifecycle is covered via an decentralized approach, complemented with private communications which is common used in the ecosystem.
+
+
+
+In this case, the following movement of tokens are done as the process progresses:
+
+* Upon launch of the payout request, the appropriate amount of funds are placed on a hold with no notary (i.e. it is an internal hold that cannot be released), and the payout is placed into a ```Ordered``` state
+* The operator then can put the payout request ```InProcess```, which prevents the _orderer_ of the payout from being able to cancel the payout request
+* After checking the payout is actually possible the operator then executes the hold, which moves the funds to a suspense wallet and places the payout into the ```FundsInSuspense``` state
+* The operator then moves the funds offchain (usually from the omnibus account)  to the appropriate destination account, then burning the tokens from the suspense wallet and rendering the payout into the ```Executed``` state
+* Either before or after placing the request ```InProcess```, the operator can also reject the payout, which returns the funds to the payer and eliminates the hold. The resulting end state of the payout is ```Rejected```
+* When the payout is ```Ordered``` and before the operator places it into the ```InProcess``` state, the orderer of the payout can also cancel it, which frees up the hold and puts the payout into the final ```Cancelled``` state
+
+
+## Specification
+
+```solidity
+interface IPayoutable /* is ERC-20 */ {
+    enum PayoutStatusCode {
+        Nonexistent,
+        Ordered,
+        InProcess,
+        FundsInSuspense,
+        Executed,
+        Rejected,
+        Cancelled
+    }
+    function approveToOrderPayout(address orderer) external returns (bool);
```

Comment:
To have the same naming as in ERC-777 I would change:
`approveToOrderPayout` -> `authorizePayoutOperator`
`revokeApprovalToOrderPayout` -> `revokePayoutOperator`
`isApprovedToOrderPayout` -> `isPayoutOperatorFor(operator, walletToDebit)` the order of the parameters is changed here